pkg/satellite/satellitedb/projectmembers_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+	"testing"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/internal/testcontext"
+	stlt "storj.io/storj/pkg/satellite"
+)
+
+func TestProjectMembersRepository(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	// creating in-memory db and opening connection
+	db, err := New("sqlite3", "file::memory:?mode=memory&cache=shared")
+	if err != nil {
+		assert.NoError(t, err)
+	}
+	defer ctx.Check(db.Close)
+
+	// creating tables
+	err = db.CreateTables()
+	if err != nil {
+		assert.NoError(t, err)
```

Comment:
Use `t.Fatal(err)` here, it shouldn't continue when this fails.

Reply:
Another way would be:
```go
if !assert.NoError(t, err) {
    return
}
```